app.py

```diff
@@ -1,25 +1,38 @@
+import os
+
 import connexion
 from config.env import env
 from connexion.resolver import MethodViewResolver
 from flask import Flask
 from openapi.utils import get_bundled_specs
+from utils.definitions import get_project_root
+
+
+project_root_path = str(get_project_root())
 
 
 def create_app(testing=False) -> Flask:
-    connexion_app = connexion.FlaskApp(__name__, specification_dir="openapi/")
+
+    options = {
+        "swagger_url": "/",
+    }
+
+    connexion_app = connexion.FlaskApp(
+        __name__, specification_dir="openapi/", options=options
+    )
 
     flask_app = connexion_app.app
 
-    if testing:
+    if (os.environ.get("FLASK_ENV") == "development") | testing:
         flask_app.config.from_object(
             "config.environments.unit_testing.UnitTestingConfig"
         )
+        from config.environments.development import DevelopmentConfig
```

Comment:
🔧 These configs do not match, should they?

Reply:
💯 that's sorted now and import is both from UnitTestingConfig 👍